tests/suites/test_suite_psa_crypto.function

```diff
@@ -1799,11 +1799,19 @@ void derive_key_policy( int policy_usage,
     PSA_ASSERT( psa_import_key( &attributes, key_data->x, key_data->len,
                                 &handle ) );
 
-    status = psa_key_derivation( &operation, handle,
-                                 exercise_alg,
-                                 NULL, 0,
-                                 NULL, 0,
-                                 1 );
+    PSA_ASSERT( psa_key_derivation_setup( &operation, exercise_alg ) );
+
+    if( PSA_ALG_IS_TLS12_PRF( exercise_alg ) ||
+            PSA_ALG_IS_TLS12_PSK_TO_MS( exercise_alg ) )
```

Comment:
Please use curly brackets for multiline operations after an if.

Reply:
I am sorry, I am probably missing something here, could you please clarify?

Reply:
Sure. I'm not sure if it's written down anywhere, but I believe that if the operation that is to be performed after an if is multi-line, curly brackets are preferred to maintain visibility. See here: https://github.com/ARMmbed/mbed-crypto/blob/54b8eabd76ddca2207d094f4d7d24398f016486c/library/psa_crypto.c#L3462

Reply:
Didn't know that. Thanks, it is indeed more readable like that.